utils/offchain_voting.js

```diff
@@ -367,32 +366,36 @@ function prepareProposalPayload(payload) {
 
 function toStepNode(step, verifyingContract, actionId, chainId, merkleTree) {
   return {
-    account: step.account,
     nbNo: step.nbNo,
     nbYes: step.nbYes,
     index: step.index,
     choice: step.choice,
     sig: step.sig,
     timestamp: step.timestamp,
-    proposalHash: step.proposalHash,
+    proposalId: step.proposalId,
+    member: step.member,
     proof: merkleTree.getHexProof(
       buildVoteLeafHashForMerkleTree(step, verifyingContract, actionId, chainId)
     ),
   };
 }
 
-async function createVote(proposalHash, account, voteYes) {
+async function createVote(proposalId, weight, voteYes) {
   const payload = {
```

Comment:
I'll simplify this in `snapshot-js-erc712` as I cannot find anywhere we use, or rely upon, `payload`.